client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/index.tsx

```diff
@@ -22,6 +26,36 @@ const thankYouContentGetter: Record< DomainThankYouType, DomainThankYouPropsGett
 
 export default thankYouContentGetter;
 
+interface StepCTAProps {
+	domainType: DomainThankYouType;
+	email?: string;
+	primary: boolean;
+}
+
+const StepCTA: FunctionComponent< StepCTAProps > = ( { email, primary, domainType } ) => {
```

Comment:
Nit: React no longer recommends `FunctionComponent` and I also find it simpler to use a function with a typed return value.
```suggestion
const StepCTA = ( { email, primary, domainType }: StepCTAProps ): JSX.Element => {
```

Reply:
Thanks for the heads up.